app/assets/javascripts/discourse/components/composer-editor.js.es6

```diff
@@ -42,6 +50,11 @@ export default Ember.Component.extend({
     return showPreview ? I18n.t('composer.hide_preview') : I18n.t('composer.show_preview');
   },
 
+  @observes('showPreview')
```

Comment:
This resulted in a failing test. Fixed via: https://github.com/discourse/discourse/commit/3fb1c21dd5fbc17fda9e4c75953d19ed8009798e

:)